### ISSUE TITLE
the motivation of the two part

### ISSUE THREAD

**Issue by seacj** [TaoRuijie/ECAPATDNN#5]
I noticed that there're two parts modification here:

> 1. self.attention = nn.Sequential(
>             nn.Conv1d(96, 128, kernel_size=1),
>             nn.ReLU(),
>             nn.BatchNorm1d(128),
>             nn.Tanh(),  # I add this layer
>             nn.Conv1d(128, 96, kernel_size=1),
>             nn.Softmax(dim=2),
>             )

> 2. self.se = nn.Sequential(
>             nn.AdaptiveAvgPool1d(1),
>             nn.Conv1d(channels, bottleneck, kernel_size=1, padding=0),
>             nn.ReLU(),
>             # nn.BatchNorm1d(bottleneck), # I remove this layer
>             nn.Conv1d(bottleneck, channels, kernel_size=1, padding=0),
>             nn.Sigmoid(),
>         )

What's the motivation here? And do these benefits the performance?
Thanks

**Comment by RJ3801** [comment]
The code is based on the ECAPA-model in VoxCeleb-Trianer

In the first part, I read the paper and the code from others, they used tanh. So I add it.

In the second part, I guess that is a mistake, I search multiple codes and find that there is no batchnorm in SE layer. So I remove it.

It can make the result better by a small margin. But I did not do specific experiments for these two settings. I do these modifications to make the code be closer to the original paper.

**Comment by TaoRuijie** [comment]
The code is based on the ECAPA-model in VoxCeleb-Trianer

In the first part, I read the paper and the code from others, they used tanh. So I add it.

In the second part, I guess that is a mistake, I search multiple codes and find that there is no batchnorm in SE layer. So I remove it.

It can make the result better by a small margin. But I did not do specific experiments for these two settings. I do these modifications to make the code be closer to the original paper.
